config/configgrpc/configgrpc.go

```diff
@@ -150,9 +150,7 @@ type ServerConfig struct {
 	IncludeMetadata bool `mapstructure:"include_metadata"`
 }
 
-// SanitizedEndpoint strips the prefix of either http:// or https:// from configgrpc.ClientConfig.Endpoint.
-// Deprecated: [v0.97.0]
```

Comment:
Since the function still exists, we can keep the comment (after lowercasing the function name and removing the deprecation), can't we?

Reply:
Added the comment back